python_src/src/lamp_py/tableau/__init__.py

```diff
@@ -1 +1,21 @@
-"""Utilites for Interacting with Tableau and Hyper files"""
+"""Utilities for Interacting with Tableau and Hyper files"""
```

Comment:
update the tableau init file to catch exceptions when the tableau modues arent found.